numpy/add_newdocs.py

```diff
@@ -5874,7 +5880,19 @@ def luf(lamdaexpr, *args, **kwargs):
     >>> np.add.reduce(X, 2)
     array([[ 1,  5],
            [ 9, 13]])
-
+           
+    You can use the initializer argument to initialize the reduction to a
+    different value. It can be used to calculate empty reductions for
+    ufuncs that don't support it as well.
```

Comment:
Move this second sentence to just above `np.minimum.reduce` so it's clear what it refers to

Reply:
Done!